src/util/initEvents.ts

```diff
@@ -20,6 +22,10 @@ export const initEvents = () => {
     store.dispatch({ type: 'setCursor', thoughtsRanked, replaceContextViews: contextViews })
   })
 
+  window.addEventListener('mousemove', _.throttle(() => {
+    store.dispatch(toggleToolbarAndBreadCrumbs(true))
+  }, 1000, { leading: true }))
```

Comment:
Only dispatch if toolbar is hidden to prevent spamming the action history.

Reply:
It's being handled in the [action-creator](https://github.com/cybersemics/em/blob/9c863f46910b550f679c250b65bb81a416330da8/src/action-creators/toggleToolbarAndBreadCrumbs.ts#L11)